lib/matplotlib/animation.py

```diff
@@ -123,7 +137,7 @@ def __init__(self, fps=5, codec=None, bitrate=None, extra_args=None,
         extra_args: list of strings or None
             A list of extra string arguments to be passed to the underlying
             movie utiltiy. The default is None, which passes the additional
-            argurments in the 'animation.extra_args' rcParam.
+            argurments in the `self.args_key` rcParam.
```

Comment:
btw, what I mean by my comment that it isn't an rcParam is that one could read it like that the rcParam is called "self.args_key", which doesn't make sense. This is going to need some rewording to be clearer.